otherlibs/stdune/src/io.ml

```diff
@@ -73,7 +73,7 @@ let copy_channels =
   loop
 
 let setup_copy ?(chmod = Fun.id) ~src ~dst () =
-  let ic = open_in src in
+  let ic = Stdlib.open_in_bin src in
```

Comment:
why does this need the `Stdlib.` qualifier?

Reply:
It doesn't, but that's the style in the rest of the file (I guess to emphasize the use of the standard library as opposed to other functions from `stdune` itself).